doc/theory/glossary.tex

```diff
@@ -185,19 +185,18 @@
 	$$
 	\paragraph{Reference:} \cite{kerber2017geometry}
 	
-	\subsection*{Persistence  landscape:}
-	A \textit{persistence  landscape} is a continuous function
+	\subsection*{Persistence landscape:}
+	A \textit{persistence landscape} is a continuous function
 	\begin{equation*}
 	\lambda : \mathbb N \times  \mathbb R \to \mathbb R \cup \{+\infty\}
 	\end{equation*}
 	and the function $\lambda_k(s) = \lambda(k,s)$ is refered to as the \textit{$k$-layer of the persistence diagram}.
 	
-	Let $\{(b_i, d_i)\}_{i \in I}$ be the persistence diagram of a \underline{persistence module}. Its \textit{associated persistence landscape} $\lambda$ is defined by letting
-	$\lambda_k(t)$ be  $k$-th largest value of 
-	$$
-	\min_{i \in I}\{t-b_i, d_i-t\}_+
-	$$
-	where $c_+$ denotes $max(c,0)$.
+	Let $\{(b_i, d_i)\}_{i \in I}$ be the persistence diagram of a \underline{persistence module}. Its \textit{associated persistence landscape} $\lambda$ is defined by letting $\lambda_k(t)$ be $k$-th largest value of the collection $\{ \Lambda_i(t) \}_{i \in I}$, where
```

Comment:
I would change this to: (using hyperref and adding a missing "the")
Let $\{(b_i, d_i)\}_{i \in I}$ be a \hyperref[persistence diagram] {persistence diagram}. Its \textit{associated persistence landscape} $\lambda$ is defined by letting $\lambda_k(t)$ be the $k$-th largest value of the collection $\{ \Lambda_i(t) \}_{i \in I}$, where

Reply:
Fixed.